build.rs

```diff
@@ -23,28 +23,79 @@
 
 extern crate gcc;
 
+use std::env;
+use std::ffi::OsString;
+use std::path::PathBuf;
+
+#[cfg(target_os = "macos")]
+const OS: &'static str = "darwin";
+#[cfg(target_os = "linux")]
+const OS: &'static str = "linux";
+#[cfg(target_os = "windows")]
+const OS: &'static str = "windows";
+
+const ANDROID_INCLUDE: &'static str = "platforms/android-21/arch-arm64/usr/include";
+
+fn android_aarch_compiler() -> String {
+	"toolchains/aarch64-linux-android-4.9/prebuilt/".to_owned() + OS + "-x86_64/bin"
+}
+
+fn android_arm_compiler() -> String {
+	"toolchains/arm-linux-androideabi-4.9/prebuilt/".to_owned() + OS + "-x86_64/bin"
```

Comment:
I disagree. Also it's less efficient, cause it needs to evaluate formatted expression